sprintly/sprintly-kanban
# Cache busting

#### What does it do?

* Fixes cache-related problems we've been having loading the main js payload
* Deals with the wercker deploy issues we've been having
  * By giving up and adding a manual deploy `make` command
  * Update us to require npm@3 for building the shrinkwrap file, hopefully squelching the back-compat 2/3 shrinkwrap issues we've been having

#### How should this be manually tested?

The javascript should have a version number appended to the file, or use the "cache" key from the package.json file when present.

_For deploys the cache key is set by running `npm run build-production`_

#### Any background context?

Not really much to see here, just a lot of trial and error to get the builds passing consistently. It's a shame that deploys from wercker are not yet fixed.

![sadsonic](https://cloud.githubusercontent.com/assets/84644/10206290/f706cc26-677a-11e5-967c-0036fbcb08d1.jpg)
